Source/Extensions/Blazorise.TreeView/Internal/_TreeViewNode.razor.cs

```diff
@@ -44,16 +44,9 @@ private async Task ToggleNode( TreeViewNodeState<TNode> nodeState, bool refresh
 
         if ( nodeState.Expanded )
         {
-            var nodeAdded = false;
-
             if ( !ExpandedNodes.Contains( nodeState.Node ) )
             {
-                nodeAdded = true;
                 ExpandedNodes.Add( nodeState.Node );
-            }
-
-            if ( nodeAdded )
-            {
                 await ExpandedNodesChanged.InvokeAsync( ExpandedNodes );
             }
```

Comment:
I'm super confused, this went back to the original version no? What I mean is 
```
if ( !ExpandedNodes.Contains( nodeState.Node ) )
            {
                ExpandedNodes.Add( nodeState.Node );
                await ExpandedNodesChanged.InvokeAsync( ExpandedNodes );
            }
```

instead of

```
if ( !ExpandedNodes.Contains( nodeState.Node ) )
            {
                ExpandedNodes.Add( nodeState.Node );
            }
            await ExpandedNodesChanged.InvokeAsync( ExpandedNodes );
```

Reply:
> I'm super confused, this went back to the original version no? What I mean is
> 
> ```
> if ( !ExpandedNodes.Contains( nodeState.Node ) )
>             {
>                 ExpandedNodes.Add( nodeState.Node );
>                 await ExpandedNodesChanged.InvokeAsync( ExpandedNodes );
>             }
> ```
> 
> instead of
> 
> ```
> if ( !ExpandedNodes.Contains( nodeState.Node ) )
>             {
>                 ExpandedNodes.Add( nodeState.Node );
>             }
>             await ExpandedNodesChanged.InvokeAsync( ExpandedNodes );
> ```

I did that

![image](https://user-images.githubusercontent.com/900302/211732872-bb9d544f-ea89-4c2c-ae97-55666a710e9a.png)